lib/PuppeteerSharp/Page.cs

```diff
@@ -696,6 +697,17 @@ public Task ExposeFunctionAsync<TResult>(string name, Func<TResult> puppeteerFun
             return request?.Response;
         }
 
+        /// <summary>
```

Comment:
this is a utility method, it will allow use to call GoToAsync like this:
```cs
page.GoToAsync("https://github.com", timeout: 3000);
```
instead of
```cs
page.GoToAsync("https://github.com", new NavigationOptions { Timeout = 3000 });
```